torch_xla/_internal/pjrt.py

```diff
@@ -96,7 +97,9 @@ def _run_singleprocess(fn: Callable[..., R], *args, **kwargs) -> Dict[int, R]:
   """
   os.environ.setdefault(xenv.PJRT_LOCAL_PROCESS_COUNT, '1')
 
-  if runtime.device_type() == 'TPU':
+  if plugins.using_dynamic_plugins():
+    plugins.default().configure_single_process()
```

Comment:
should we throw a warning or something when pople configure `PJRT_DEVICE` while also register the plugin in the code?

Reply:
You still select the device type with `PJRT_DEVICE`. Plugins will just let you register new device types when we clean up all of the hardcoded strings.